libraries/libfc/include/fc/io/random_access_file.hpp

```diff
@@ -0,0 +1,465 @@
+#pragma once
+#include <fc/filesystem.hpp>
+#include <fc/io/datastream.hpp>
+#include <fc/io/raw.hpp>
+
+#include <boost/beast.hpp>
+
+#include <boost/iostreams/categories.hpp>
+#include <boost/iostreams/pipeline.hpp>
+#include <boost/iostreams/positioning.hpp>
+#include <boost/interprocess/mapped_region.hpp>
```

Comment:
It's needed due to use of some boost interprocess types such as
https://github.com/AntelopeIO/spring/blob/a29f3f96aa9bfebc3a33eb4c3578bab74bb08527/libraries/libfc/include/fc/io/random_access_file.hpp#L448
`nodeos` will build with the include removed because we're probably picking up some chainbase stuff somewhere before `random_access_file.hpp` is included, but if you remove this include `test_fc` def doesn't compile because of missing `boost::interprocess::mapping_handle_t`